.travis.yml

```diff
@@ -1,9 +1,33 @@
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
+  - "2.7.12"
+  - "3.5"
+
+install:
+  - pip install -r requirements.txt
+  - pip install flake8
+
+before_script:
+  # --exit-zero means that flake8 will only warn and not stop the build
```

Comment:
Nice idea.  I will create a `.flake8` file at root.  I do not know how to make "git commit hooks" but would be fascinated to learn.